src/components/IndexPage/testimonial.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import PropTypes from "prop-types"
+import Body from "styles/Body"
+import { HeaderLineBelow } from "styles/Header"
+import Section from "styles/Section"
+import styled from "styled-components"
+
+const SectionPad = styled(Section)`
+  padding-top: 0px;
+`
+
+const Content = styled.div`
+  max-width: 600px;
+  margin: 0px auto;
+`
+
+const AuthorInfo = styled(Body)`
+  text-align: right;
+  font-style: italic;
+`
+
+const Testimonial = ({ quote, author, role }) => (
+  <SectionPad>
+    <HeaderLineBelow>Work with us</HeaderLineBelow>
```

Comment:
We might not need to have a header here. What do you think about that idea?

Reply:
check it out. idk if it's better